### DISCUSSION TITLE
How to add markdown link to image?

### DISCUSSION THREAD

**Discussion by OCram85** [vuejs/vitepress#3357] [Q&A] [answered]
Just noticed that links to images doesn't work for  `vitepress preview` / production deployment.

```markdown
[![img](./path-to-img.excalidraw.png)](./path-to-img.excalidraw.png)
```

Is there a way to use the image like this?  I guess it's caused by vite / bundler because it uses the `assets/` folder and adds an id to the file names.

**Comment by brc-dd** [comment, marked as answer]
move image to public directory 👀 ?

**Reply by OCram85** [reply]
Awesome that works! 👍 🎉 
So now I guess the only downside is, I have to organize the images in the public directory manually 😄 .
